CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
 # Acra ChangeLog
 
-## [0.92.0](https://github.com/cossacklabs/acra/releases/tag/0.92.0), March 01th 2021
+## [0.93.0](https://github.com/cossacklabs/acra/releases/tag/0.93.0), May 27th 2021
+
+_Core_:
+
+- **AcraServer**:
+  - Added masking real binary data type from database for application and mask it with application's types on database protocol layer. Extended encryptor_config which allow configure mapping application data type to proper database's type. [#515](https://github.com/cossacklabs/acra/pull/515), [#517](https://github.com/cossacklabs/acra/pull/517), [#523](https://github.com/cossacklabs/acra/pull/523), [#519](https://github.com/cossacklabs/acra/pull/519), [#520](https://github.com/cossacklabs/acra/pull/520)
+  - Extended `encryptor_config` with new parameters:
+    - `data_type` - specify data type expected by application. Accept `str`, `bytes`, `int64`, `int32` values. [#515](https://github.com/cossacklabs/acra/pull/515), [#517](https://github.com/cossacklabs/acra/pull/517)
+    - `default_data_value` - specify default value to replace not decrypted data. [#515](https://github.com/cossacklabs/acra/pull/515), [#517](https://github.com/cossacklabs/acra/pull/517)
+    - `response_on_fail` - specify action on decryption failure. Accept `ciphertext` (returns encrypted data as is), `default_value` (returns values from `default_data_value` parameter), `error` (returns error as DB error with message like `encoding error in column {column_name}`). [#521](https://github.com/cossacklabs/acra/pull/521), [#533](https://github.com/cossacklabs/acra/pull/533)
+  - Has been deprecated `tokenize` parameter in `encryptor_config` and changed focus on `token_type` parameter. Now is enough to specify `token_type` parameter without `tokenize: true` to turn on tokenization. [Read more](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config/#tokenized-deprecated-since-0930) in the documentation. [#527](https://github.com/cossacklabs/acra/pull/527)
```

Comment:
```suggestion
  -  Deprecated `tokenize` parameter in `encryptor_config` and changed focus on `token_type` parameter. Now is enough to specify `token_type` parameter without `tokenize: true` to turn on tokenization. [Read more](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config/#tokenized-deprecated-since-0930) in the documentation. [#527](https://github.com/cossacklabs/acra/pull/527)
```